install/kubernetes/helm/istio/values.yaml

```diff
@@ -307,6 +307,7 @@ pilot:
   enabled: true
   replicaCount: 1
   image: pilot
+  traceSampling: 10.0
```

Comment:
just curious: any particular reason to choose this as a default?

Reply:
Just picked something reasonable.